.github/workflows/ci.yaml

```diff
@@ -80,6 +80,71 @@ jobs:
           OLLAMA_MODEL: ${{ matrix.ollama-model }}
 
       - name: Run Ollama tests
-        run: uv run pytest tests -m integration -k ollama
+        run: uv run pytest tests -m integration -k ollama || cat ollama.log
```

Comment:
IIUC `cat ollama.log` will be a successful status code in the case that tests pass, should it be `(cat ollama.log && exit 1)`? Sorry if I just misunderstood something

Reply:
yeah.. probably right everywhere